### DISCUSSION TITLE
Confirmation E-mail

### DISCUSSION THREAD

**Discussion by jperica** [teaxyz/white-paper#122] [Q&A]
How long does it take for confirmation email to arrive?

**Comment by thomas-borrel** [comment]
Let us know if you still haven't received the confirmation email.

**Comment by Rembranced** [comment]
We will have to wait

**Comment by jperica** [comment]
> Let us know if you still haven't received the confirmation email.

I stil did not get email. its been 48 hours.

**Comment by thomas-borrel** [comment]
Thank you for confirming - Looking into it.

**Comment by jperica** [comment]
I used gmail.com and I got it.

**Comment by jperica** [comment]
but I did not get it on the original email.

**Reply by thomas-borrel** [reply]
Thanks for the update. The team is looking into it. Glad you could sign-up though. I assume you check the spam folder, etc... of the original email?

**Reply by 0xsingular** [reply]
@thomas-borrel If it helps, most people using gmail have this problem and get the confirmation email in 2-4 days, people using hotmail or outlook get the confirmation email in 2 minutes.

**Reply by thomas-borrel** [reply]
Thanks for sharing @0xsingular - We need to understand why that is.

**Reply by jperica** [reply]
I got the email on original email after 3 days. I got it on gmail after 2 mins and both not in the spam.

**Reply by 0xsingular** [reply]
From which provider is your “original email”?

**Reply by jperica** [reply]
@sentanel.com but I use gmail for that emails as well

**Comment by rav2525** [comment]
I have used gmail and received sign up confirmation almost instantly

**Reply by jperica** [reply]
yes with Gmail I did not have any problems.

**Comment by 0xsingular** [comment]
The issue has been fixed!

**Comment by hamaadh** [comment]
> How long does it take for confirmation email to arrive?

No specific time, I received within 24 hour، try to submit again.
